packages/KMenu/KMenuItem.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div
+    :data-testid="item ? `${item.title.replace(' ', '-')}-menu-item` : 'menu-item'"
+    :test-mode="testMode"
+    :class="[isOpen && expandable ? 'title-dark' : '', {'expand-item' : expandable}]"
+    class="k-menu-item">
+    <div
+      class="menu-title"
+      @click="isOpen = !isOpen">
+      <span
+        :class="isOpen && expandable ? 'title-dark' : ''"
+        class="span-menu-title">
+        <slot name="itemTitle">
+          {{ item ? item.title : '' }}
+        </slot>
+      </span>
+      <span
+        class="span-icon-container">
+        <KIcon
+          v-if="expandable"
+          :icon="isOpen ? 'chevronUp' : 'chevronDown'"
+          color="var(--grey-400)"
+          size="16"
+        />
+      </span>
+    </div>
+    <div
```

Comment:
can we add aria attributes for this

aria-labelledby

Reply:
Done